components/feature/tab-collections/build.gradle

```diff
@@ -49,8 +49,7 @@ dependencies {
     implementation Dependencies.kotlin_coroutines
 
     implementation Dependencies.androidx_paging
-    implementation Dependencies.androidx_lifecycle_extensions
-    kapt Dependencies.androidx_lifecycle_compiler
+    implementation Dependencies.androidx_lifecycle_livedata
```

Comment:
🤔 What do we need `livedata` dependency for?

Reply:
`androidx_lifecycle_extensions` had both LiveData and ViewModel and was replaced with two separate dependencies. `TabCollectionDao` currently uses `LiveData` to return some results so I left a dependency on `androidx_lifecycle_livedata`.